src/Sarif.Converters/ToolFormat.cs

```diff
@@ -31,7 +31,7 @@ public static class ToolFormat
         public const string PREfast = nameof(PREfast);
 
         /// <summary>Semmle's file format.</summary>
-        public const string SemmleQL = nameof(SemmleQL);
+        public const string SemmleQl = nameof(SemmleQl);
```

Comment:
Fixed.

---
In reply to: [121228603](https://github.com/Microsoft/sarif-sdk/pull/582#discussion_r121228603) [](ancestors = 121228603)